ui/components/component-library/avatar-network/avatar-network.types.ts

```diff
@@ -0,0 +1,43 @@
+import type { PolymorphicComponentPropWithRef } from '../box';
+import type { AvatarBaseStyleUtilityProps } from '../avatar-base/avatar-base.types';
+import { AvatarBaseSize } from '../avatar-base/avatar-base.types';
+
+export enum AvatarNetworkSize {
+  Xs = AvatarBaseSize.Xs,
+  Sm = AvatarBaseSize.Sm,
+  Md = AvatarBaseSize.Md,
+  Lg = AvatarBaseSize.Lg,
+  Xl = AvatarBaseSize.Xl,
```

Comment:
Unfortunately, we can't reassign enums we have to define them again
```suggestion
  Xs = 'xs',
  Sm = 'sm',
  Md = 'md',
  Lg = 'lg',
  Xl = 'xl',
```